src/include/ceph_fs.h

```diff
@@ -422,7 +422,8 @@ enum {
 	CEPH_MDS_OP_ENQUEUE_SCRUB  = 0x01503,
 	CEPH_MDS_OP_REPAIR_FRAGSTATS = 0x01504,
 	CEPH_MDS_OP_REPAIR_INODESTATS = 0x01505,
-	CEPH_MDS_OP_RDLOCK_FRAGSSTATS = 0x01507
+	CEPH_MDS_OP_RDLOCK_FRAGSSTATS = 0x01507,
+	CEPH_MDS_OP_DUMMY = 0x01600
```

Comment:
You may need to add a dummy reclaim flag to test this too.

Reply:
Okay

Reply:
As we decided not to evict client in case of an unknown reclaim flag, I.e. keeping the code as it is with a minor change of  the type of error to be `CEPHFS_EINVAL` instead of `CEPHFS_EOPNOTSUPP`, im closing this conversation

Reply:
Move this below `CEPH_MDS_OP_GETVXATTR`, and set to: `0x00107`

Reply:
Move this below CEPH_MDS_OP_GETVXATTR, and set to: 0x00107

> this is done